common/lib/dependabot/pull_request_creator/azure.rb

```diff
@@ -10,6 +10,11 @@ class Azure
                   :files, :commit_message, :pr_description, :pr_name,
                   :author_details, :labeler, :reviewers, :assignees, :work_item
 
+      MAX_PR_DESCRIPTION_LENGTH = 3999 # Azure has a max length of 4000 (0 based count)
+      ENCODING = Encoding::UTF_16 # Azure DevOps only support descriptions up to 4000 characters in UTF-16
+      # encoding.
+      # https://developercommunity.visualstudio.com/content/problem/608770/remove-4000-character-limit-on-pull-request-descri.html
```

Comment:
```suggestion
      # Azure DevOps has a max description length of 4,000 characters in UTF-16 encoding:
      # https://developercommunity.visualstudio.com/content/problem/608770/remove-4000-character-limit-on-pull-request-descri.html
      PR_DESCRIPTION_MAX_LENGTH = 3_999 # zero-based indexing
      PR_DESCRIPTION_ENCODING = Encoding::UTF_16
```

The slight renames of the constants was intentional for improved clarity and consistency... for example encoding could be different for PR description vs other bits like titles, labels, etc...